src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +23,28 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    public LineResponse updateLine(Long id, LineRequest request) {
+        Line persistLine = lineRepository.findById(id).get();
+        persistLine.update(request.toLine());
+        lineRepository.save(persistLine);
+        return LineResponse.of(persistLine);
+    }
+
+    public List<LineResponse> findAllLines() {
+        List<Line> lines = lineRepository.findAll();
+
+        return lines.stream()
+            .map(line -> LineResponse.of(line))
```

Comment:
```suggestion
            .map(LineResponse::of)
```
람다의 메서드 참조를 사용하면 더 깔끔할 것 같아요 !

Reply:
넵. 가능하면 람다식으로 간소화하겠습니다.